uxarray/grid/intersections.py

```diff
@@ -7,6 +7,48 @@
 from uxarray.utils.computing import cross_fma, allclose, dot, cross, norm
 
 
+from numba import njit, prange
+
+
+@njit(parallel=True, nogil=True, cache=True)
+def fast_constant_lat_intersections(lat, edge_node_z, n_edge):
```

Comment:
A bit skeptical about having "fast" in the function name; could we find an alternative way to emphasize likewise? Also, as before, I think I find it more intuitive to start the function name with the biggest objective of the function, e.g. something like `intersections_at_constant_latitude`?

Reply:
> A bit skeptical about having "fast" in the function name; could we find an alternative way to emphasize likewise?

Since this function will be paired alongside @hongyuchen1030 accurate algorithm, there is a need to emphasize the difference. I was envisioning:
* `fast_constant_lat_intersections`
* `accurate_constant_lat_intersections`

Open to any other ideas though.

Reply:
I was expecting to see the distinction between algorithms being exphasized through an argument in a single function signature, rather than having separate functions.

Reply:
For instance `intersections_at_constant_latitude(method=`fast`)`. Fast or approximate for the first, and accurate for the second would make sense. @hongyuchen1030 what do you also think?

Reply:
> I was expecting to see the distinction between algorithms being exphasized through an argument in a single function signature, rather than having separate functions.

This doesn't work well with Numba, which is the main reason I prefixed the function.

Reply:
I see it better now; thanks for clarifying!